From: "Steven Rostedt (Red Hat)" <rostedt@goodmis.org>
Date: Fri, 26 Feb 2016 10:03:48 -0500
Subject: [PATCH 379/397] rtmutex: Have slowfn of rt_mutex_timed_fastlock() use
 enum
Origin: https://git.kernel.org/cgit/linux/kernel/git/rt/linux-stable-rt.git/commit?id=625c22a30085f3ca42642b759673e6595ce39493

The backport of commit 8930ed80f970 "rtmutex: Cleanup deadlock detector
debug logic" had conflicts, and the conflict resolution changed
rt_mutex_timed_fastlock()'s parameter to an enum, but missed changing its
slowfn's prototype.

Signed-off-by: Steven Rostedt <rostedt@goodmis.org>
---
 kernel/rtmutex.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/rtmutex.c b/kernel/rtmutex.c
index 2a4c34bb586f..e836b05093a1 100644
--- a/kernel/rtmutex.c
+++ b/kernel/rtmutex.c
@@ -1657,7 +1657,7 @@ rt_mutex_timed_fastlock(struct rt_mutex *lock, int state,
 			enum rtmutex_chainwalk chwalk,
 			int (*slowfn)(struct rt_mutex *lock, int state,
 				      struct hrtimer_sleeper *timeout,
-				      int detect_deadlock))
+				      enum rtmutex_chainwalk chwalk))
 {
 	if (chwalk == RT_MUTEX_MIN_CHAINWALK &&
 	    likely(rt_mutex_cmpxchg(lock, NULL, current))) {
